frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature

```diff
@@ -45,18 +45,18 @@ Feature: Create workload from Operator Backed file
               And user is able to see "jaeger-all-in-one-inmemory1" in Topology page
 
 
-        @regression @to-do @odc-6467
+        @regression @odc-6467
         Scenario: Bindable resource in Operator Backed: A-08-TC05
             Given user has installed Service Binding operator
               And user has installed Crunchy Postgres for Kubernetes operator
               And user has created or selected namespace "aut-service-binding"
               And user is at OperatorBacked page
              When user enters "Postgres Cluster" in Filter by keyword
-             Then user will see "Bindable" label associated with "Postgres Cluster" card
-              And user will see "Bindable" label in "Postgres Cluster" sidebar
+             Then user will see Bindable label associated with catalog card
```

Comment:
Why is the Bindable label removed from the quotes, you can use it with `contains`  using variable